### ISSUE TITLE
Switch to pymc3_ext for sampling

### ISSUE THREAD

**Issue by dfm** [dfm/tess-atlas#47]
I've found that the following works much better than the current sampling method. Give it a shot!

```python
        trace = pmx.sample(
            tune=TUNE,
            draws=DRAWS,
            start=init_params,
            chains=2,
            cores=2,
        )
```

I'm also finding that the parallel chains work fine now. You?

**Comment by dfm** [comment]
Ref: #30

**Comment by avivajpeyi** [comment]
Haven't tried messing around with the number of chains, but will try it out!

**Comment by dfm** [comment]
Fixed in #51
